.gitattributes

```diff
@@ -0,0 +1 @@
+*.cabal linguist-generated=true
```

Comment:
This prevents GitHub from showing those generated files by default when doing code reviews.